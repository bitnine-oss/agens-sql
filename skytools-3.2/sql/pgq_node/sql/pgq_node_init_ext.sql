
create extension pgq;

\set ECHO none
\i structure/install.sql
\set ECHO all
create extension pgq_node from unpackaged;
select array_length(extconfig, 1) as dumpable from pg_catalog.pg_extension where extname = 'pgq_node';
drop extension pgq_node;

create extension pgq_node;
select array_length(extconfig, 1) as dumpable from pg_catalog.pg_extension where extname = 'pgq_node';

