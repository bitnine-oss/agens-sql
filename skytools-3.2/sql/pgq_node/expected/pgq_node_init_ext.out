create extension pgq;
\set ECHO none
 upgrade_schema 
----------------
              0
(1 row)

create extension pgq_node from unpackaged;
select array_length(extconfig, 1) as dumpable from pg_catalog.pg_extension where extname = 'pgq_node';
 dumpable 
----------
        4
(1 row)

drop extension pgq_node;
create extension pgq_node;
select array_length(extconfig, 1) as dumpable from pg_catalog.pg_extension where extname = 'pgq_node';
 dumpable 
----------
        4
(1 row)

